script/cms/compare-json.js

```diff
@@ -154,6 +157,49 @@ const getParentNode = (
   return parent;
 };
 
+/**
+ * Compares two arrays using lodash isEqual on each item.
+ * TODO: convert this output to match deep-diff format
+ * @param cmsExportArray
+ * @param graphQLArray
+ * @param dataPath
+ */
+const compareArrays = (cmsExportArray, graphQLArray, dataPath) => {
+  const cmsArrayLength = cmsExportArray.length;
+  const graphQLLength = graphQLArray.length;
+
+  if (cmsArrayLength !== graphQLLength) {
+    console.log(
+      'ARRAY LENGTHS DIFFER FOR',
+      dataPath,
+      'cmsExport length: ',
+      cmsArrayLength,
+      'graphQL length',
+      graphQLLength,
+    );
+  }
+
+  cmsExportArray.forEach((cmsItem, index) => {
+    if (!isEqual(cmsItem, graphQLArray[index])) {
+      console.log('ARRAY DIFF FOUND FOR', dataPath, 'index', index);
+      console.log('CMS item:', JSON.stringify(cmsItem, null, 2));
```

Comment:
Can we output this to a file maybe? I imagine it will be a lot displayed to the console.

Reply:
That can be done using a UNIX redirect, if need be. So far I am not seeing any output at all. If that turns out to be erroneous I will reconsider. Ultimately we should probably put this data into the same format as the other diffs, but since I wasn't seeing anything I figured a simple console.log would suffice to unblock facilities and PW teams for now.

Reply:
To get this data to have the same format as the other results you have the `compareArrays` function return an object similar to what we return, i.e. 
```js
{
    diffType: "Array Diff",
    path: dataPath,
    graphQL: graphQLArray[index]
    cmsExport: cmsItem
}
  ```

 and call the `compareArrays` function directly in the `if (isArray)` statement, and push the results to an array. Then you can append that array to the `diffs` array if it's not `undefined`. This way, the array differences would be outputted in the same diff file.